tests/scattering1d/test_torch_scattering1d.py

```diff
@@ -439,13 +439,12 @@ def test_Q(device, backend):
         _ = Scattering1D(
             J, shape, Q=0.9, backend=backend, frontend='torch')
     assert "Q should always be >= 1" in ve.value.args[0]
-
+    
```

Comment:
remove?

Reply:
my bad. my editor (Atom) added this automatically
6f3715b